I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 284, in script
    play music quiz
  File "renpy/common/000statements.rpy", line 124, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 33, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "game/script.rpy", line 284, in <module>
    play music quiz
NameError: name 'quiz' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 284, in script
    play music quiz
  File "C:\Users\D E L L\Downloads\renpy\renpy-8.3.2-sdk\renpy\ast.py", line 1974, in execute
    self.call("execute")
  File "C:\Users\D E L L\Downloads\renpy\renpy-8.3.2-sdk\renpy\ast.py", line 1953, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\D E L L\Downloads\renpy\renpy-8.3.2-sdk\renpy\statements.py", line 374, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 124, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 33, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "C:\Users\D E L L\Downloads\renpy\renpy-8.3.2-sdk\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\D E L L\Downloads\renpy\renpy-8.3.2-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 284, in <module>
    play music quiz
NameError: name 'quiz' is not defined

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.3.24111502
Sea Savers 1.0
Thu Dec 12 22:18:16 2024
